pkg/sql/jobs_profiler_execution_details.go

```diff
@@ -244,7 +248,45 @@ func (e *executionDetailsBuilder) addDistSQLDiagram(ctx context.Context) {
 		if err := jobs.WriteExecutionDetailFile(ctx, filename,
 			[]byte(fmt.Sprintf(`<meta http-equiv="Refresh" content="0; url=%s">`, dspDiagramURL)),
 			e.db, e.jobID); err != nil {
-			log.Errorf(ctx, "failed to write DistSQL diagram for job %d: %+v", e.jobID, err.Error())
+			log.Errorf(ctx, "failed to write DistSQL diagram for job %d: %v", e.jobID, err.Error())
 		}
 	}
 }
+
+// addClusterWideTraces generates and persists a `trace.<timestamp>.zip` file
+// that captures the active tracing spans of a job on all nodes in the cluster.
+func (e *executionDetailsBuilder) addClusterWideTraces(ctx context.Context) {
+	z := zipper.MakeInternalExecutorInflightTraceZipper(e.db.Executor())
+
+	var traceID tracingpb.TraceID
+	if err := e.db.Txn(ctx, func(ctx context.Context, txn isql.Txn) error {
```

Comment:
Might be nice to extract this to a function

Reply:
good idea, done